src/azure-cli/azure/cli/command_modules/alertsmanagement/_params.py

```diff
@@ -0,0 +1,182 @@
+        c.argument('target_resource_type', target_resource_type_arg_type, required=False)
+        c.argument('target_resource_group', target_resource_group_arg_type, required=False)
+        c.argument('severity', severity_arg_type, required=False)
+        c.argument('monitor_service', monitor_service_arg_type, required=False)
+        c.argument('impacted_scope', options_list=['--impacted-scope'], required=False,
+                   help='Gets all actions rules in a subscription filter by impacted scope')
+        c.argument('description', options_list=['--description'], required=False,
+                   help='Gets all actions rules in a subscription filter by description')
+        c.argument('alert_rule', alert_rule_arg_type, required=False)
+        c.argument('action_group', options_list=['--action-group'], required=False,
+                   help='Gets all actions rules in a subscription filter by action group')
+        c.argument('name', options_list=['--name'], required=False,
+                   help='Gets all actions rules in a subscription filter by action rule name')
+
+    with self.argument_context('alertsmanagement action-rule show') as c:
+        c.argument('resource_group_name', resource_group_name_arg_type, required=True)
+        c.argument('action_rule_name', options_list=['--name'], required=True, help='Name of action rule')
+
+    with self.argument_context('alertsmanagement action-rule set') as c:
+        c.argument('resource_group_name', resource_group_name_arg_type, required=True)
+        c.argument('name', options_list=['--name'], required=True, help='Unique name for action rule')
+        c.argument('description', options_list=['--description'], required=False, help='Description of Action Rule')
+        c.argument('status', options_list=['--status'], required=True, help='Status of Action Rule')
+        c.argument('scope', options_list=['--scope'], required=True, help='Comma separated list of values')
+        c.argument('severity_condition', options_list=['--severity-condition'], required=False,
+                   help='Expected format - {<operation>:<comma separated list of values>} For eg. Equals:Sev0,Sev1')
+        c.argument('monitor_service_condition', options_list=['--monitor-service-condition'], required=False,
+                   help='Expected format - {<operation>:<comma separated list of values>}' +
+                   ' For eg. Equals:Platform,Log Analytics')
+        c.argument('monitor_condition', monitor_condition_arg_type, required=False)
+        c.argument('target_resource_type_condition', options_list=['--target-resource-type-condition'], required=False,
+                   help='Expected format - {<operation>:<comma separated list of values>}' +
+                   ' For eg. Contains:Virtual Machines,Storage Account')
+        c.argument('alert_ruleId_condition', options_list=['--alert-ruleId-condition'], required=False,
```

Comment:
could there be more than one condition?

Reply:
Conditions are mutually exclusive and there can be more than one condition